airmozilla/starred/tests/test_starred.py

```diff
@@ -0,0 +1,198 @@
+import json
```

Comment:
Yup I noted that I'd change it once I was closer to being done with it. The temporary name just keeps me less confused as I move between files.